src/components/analytics/TraceAnalytics.tsx

```diff
@@ -0,0 +1,35 @@
+import { sendAnalyticsEvent, Trace, TraceEvent } from '@uniswap/analytics'
+import { atomWithStorage, useAtomValue } from 'jotai/utils'
+import { memo } from 'react'
+
+const analyticsOptOutAtomName = 'optOutOfAnalytics'
+export const analyticsOptOutToggle = atomWithStorage<boolean>(analyticsOptOutAtomName, true)
```

Comment:
Have we used other atoms with storage, or is this a new pattern? If we're using localStorage, should this potentially go through redux instead, to avoid writing to localStorage from multiple libraries?